internal/log/log.go

```diff
@@ -113,6 +118,10 @@ func (logger *Logger) logf(severityLevel severity, format string, args ...interf
 		args = prependString(logger.prefix, args...)
 	}
 
+	format = "%s " + format
+
+	args = prependString(severityPrefix(severityLevel), args...)
+
```

Comment:
Feel free to disregard because this is a borderline/subjective case, but fwiw I think the local var name `sev` would go down easier as you parse this code, and would be ok given how short the func is.